### PR TITLE
Edits for four open issues

### LINKED ISSUE #4: profile or eat_profile?
This doc refers to both "profile" and "eat_profile". EAT itself uses both terms, the first for the CBOR claim and the second for the JSON claim (because "profile" is already registered in JWT). 

I think we either need to fix EAT so it just uses "eat_profile", or explain the mixing of terms in this document.

### LINKED ISSUE #7: one more reason for defining a profile parameter
Dave Thaler in https://mailarchive.ietf.org/arch/msg/rats/5BZdOsNNquo9l6nuUSEbkobOWiY/

```
I have just one wording suggestion.  In section 3 it says:

> The media types defined in this document include an optional profile
> parameter that can be used to mirror the eat_profile claim of the
> transported EAT.  Exposing the EAT profile at the API layer allows
> API routers to dispatch payloads directly to the profile-specific
> processor without having to snoop into the request bodies. 
 
That is of course true, but the justification can be even stronger by pointing out that it also allows use when the payload is not present in the message, such as in an Accept header (and indeed section 4 shows that).  
 
Indeed, the TEEP protocol uses it in both Content-Type and Accept, just like in section 4.
```

### LINKED ISSUE #8: DEB is now BUNDLE
DEB has been dropped in favour of BUNDLE, see https://github.com/ietf-rats-wg/eat/pull/252

We need to take care of the fallout.

### LINKED ISSUE #9: Add ref to RFC9205
Dave T's note:
```
Separately, for use in HTTP, see RFC 9205's discussion of media types.
This was recently published by the HTTPbis working group. It's possible
(I don't know) that it could be a useful Informative reference from the
eat-media-type draft.
```

### PR DESCRIPTION
**Pull request by thomas-fossati**
* rename the media-type parameter from `profile` to `eat_profile`
* DEB (the acronym) is no more
* add refs to REST design documents for HTTP and CoAP
* two edits suggested by Carl and Dave
* update references to match the latest EAT sectioning

also:

* add line RFC8792 line folding in HTTP examples
* add a changelog

Fix #4
Fix #7
Fix #8
Fix #9

Signed-off-by: Thomas Fossati <thomas.fossati@arm.com>

### CODE REVIEW

File: draft-ietf-rats-eat-media-type.md [hunk 1]
```diff
@@ -104,52 +112,60 @@ CBOR flavours), whilst {{UCCS}} defines the remaining two: UCCS and UJCS.
...
 understand in order to interoperate.  For example: the number and type of
 claims, which serialisation format, the supported signature schemes, etc.  EATs
 carry an in-band profile identifier using the `eat_profile` claim (see
-{{Section 4.3.3 of EAT}}).  The value of the `eat_profile` claim is either an
+{{Section 4.3.2 of EAT}}).  The value of the `eat_profile` claim is either an
 OID or a URI.
 
-The media types defined in this document include an optional `profile`
-parameter that can be used to mirror the `eat_profile` claim of the transported
+The media types defined in this document include an optional `eat_profile`
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat-media-type.md:124]: yes

File: draft-ietf-rats-eat-media-type.md [hunk 2]
```diff
@@ -169,8 +185,8 @@ IANA is requested to add the following media types to the
 | Name | Template | Reference |
 | EAT CWT | application/eat-cwt | {{&SELF}}, {{media-type-eat-cwt}} |
 | EAT JWT | application/eat-jwt | {{&SELF}}, {{media-type-eat-jwt}} |
-| EAT CBOR DEB | application/eat-deb+cbor | {{&SELF}}, {{media-type-deb-cbor}} |
-| EAT JSON DEB | application/eat-deb+json | {{&SELF}}, {{media-type-deb-json}} |
+| Detached EAT Bundle CBOR | application/eat-bun+cbor | {{&SELF}}, {{media-type-deb-cbor}} |
+| Detached EAT Bundle JSON | application/eat-bun+json | {{&SELF}}, {{media-type-deb-json}} |
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat-media-type.md:189]: yes

File: draft-ietf-rats-eat-media-type.md [hunk 3]
```diff
@@ -104,52 +112,60 @@ CBOR flavours), whilst {{UCCS}} defines the remaining two: UCCS and UJCS.
...
-parameter with the corresponding OID or URL.
+derived from the base (e.g., `application/eat-cwt)` by populating the
+`eat_profile` parameter with the corresponding OID or URL.
 
 # Examples
 
 The example in {{fig-rest-req}} illustrates the usage of EAT media types for
-transporting attestation evidence.
+transporting attestation evidence as well as negotiating the acceptable format
+of the attestation result.
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat-media-type.md:138]: yes

File: draft-ietf-rats-eat-media-type.md [hunk 4]
```diff
@@ -104,52 +112,60 @@ CBOR flavours), whilst {{UCCS}} defines the remaining two: UCCS and UJCS.
...
 POST /challenge-response/v1/session/1234567890 HTTP/1.1
 Host: verifier.example
-Accept: application/eat-cwt; profile="tag:ar4si.example,2021"
-Content-Type: application/eat-cwt; profile="tag:evidence.example,2022"
+Accept: application/eat-cwt; eat_profile="tag:ar4si.example,2021"
+Content-Type: application/eat-cwt; \
+              eat_profile="tag:evidence.example,2022"
 
-[ CBOR-encoded EAT w/ profile="tag:evidence.example,2022" ]
+[ CBOR-encoded EAT w/ eat_profile="tag:evidence.example,2022" ]
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat-media-type.md:149]: yes

**Review by laurencelundblade — APPROVED** [review]
I like "eat-bun" :-). Looking forward cinnamon profiles, particularly ones that prohibits raisins.

File: draft-ietf-rats-eat-media-type.md [hunk 5]
```diff
@@ -40,6 +41,9 @@ normative:
 
 informative:
   RATS-Arch: I-D.ietf-rats-architecture
+  BUILD-W-HTTP: BCP56
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat-media-type.md:44]: yes

File: draft-ietf-rats-eat-media-type.md [hunk 6]
```diff
@@ -82,7 +86,10 @@ APIs ({{fig-api-sd}}).
 
 This memo defines media types to be used for Entity Attestation Token (EAT)
 {{EAT}} payloads independently of the RATS Conceptual Message in which they
-manifest themselves.
+manifest themselves.  The objective is to give protocol, API and application
+designers a number of readily available and reusable media types for
+integrating EAT-based messages in their flows, for example when using HTTP
+{{BUILD-W-HTTP}} or CoAP {{REST-IoT}}.
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat-media-type.md:92]: yes, analogously

File: draft-ietf-rats-eat-media-type.md (hunk 1 above)
Comment (henkbirkholz) [inline draft-ietf-rats-eat-media-type.md:124]: yes

**Review by henkbirkholz — APPROVED** [review]
#4, #7, #8, #8 seem fixed, DEB is gone, refs and editorial update looks good.
Big fan of qualifying named words that would lead to collisions at some point, surely.

**Pull request merged**